notNeededPackages.json

```diff
@@ -608,6 +608,10 @@
             "libraryName": "chai-http",
             "asOfVersion": "4.2.0"
         },
+        "chai-quantifiers": {
+            "libraryName": "chai-quantifiers",
+            "asOfVersion": "1.0.12"
```

Comment:
it's a little worrisome to point the deprecation to a version that differs only in patch version. But the types have only incremented the patch version to 1.0.4, so the stub should publish at 1.0.5 and be fine.